sdk/spring/spring-cloud-azure-integration-test-appconfiguration-config/src/test/java/com/azure/spring/cloud/config/AppConfiguration.java

```diff
@@ -9,13 +12,16 @@
 @Configuration
 public class AppConfiguration {
 
+    @Value("${authMethod}")
+    private String authMethod;
```

Comment:
Can you add a few comments here and a few other places. We don't touch this often. I wouldn't want to forget why we did this.